src/main/java/Duke.java

```diff
@@ -1,10 +1,111 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.Scanner;
+
+public class Duke {
+
+    public static void printSeparator() {
+        String separator="____________________________________________________________";
+        System.out.println(separator);
+    }
+
+    public static void main(String[] args) {
+
+        showWelcomeScreen();
+
+        Scanner in = new Scanner(System.in);
+        Task[] taskArray = new Task[100];
+        int inputCount=0;
+        int outputCount=1;
+
+        String inputString = in.nextLine();
+
+        while ( !inputString.equals("bye") ) {
+            if ( !inputString.equals("list") && !inputString.contains("done ")) {
+                if (inputString.contains("todo")) {
+                    taskArray[inputCount]=new ToDo(inputString.replace("todo",""));
+                    printMessage(taskArray[inputCount]);
+                    inputCount++;
+                    System.out.println("Now you have "+inputCount+" tasks in the list.");
+                }
+
+                else if (inputString.contains("deadline")) {
+                    String[] inputDeadline=inputString.split("/");
+                    taskArray[inputCount]=new Deadline(inputDeadline[0].replace("deadline",""),
+                            inputDeadline[1].replace("by",""));
+                    printMessage(taskArray[inputCount]);

```

Comment:
I like this refactoring :) BTW, how about giving a more intuitive name such as `printSuccessfullyAddedMessage()`?